ai/commands.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import subprocess
+import sys
+
+from .network import Network
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+formatter = logging.Formatter(
+    "%(asctime)s - %(name)-11s - %(levelname)-8s - %(message)s",
+    datefmt="%H:%M:%S",
+)
+
+stream_handler = logging.StreamHandler()
+stream_handler.setFormatter(formatter)
+
+logger.addHandler(stream_handler)
```

Comment:
You can just define the logger using
```suggestion
logger = logging.getLogger(__package__)
```
and then configure it in the main function!